src/hotspot/share/gc/shenandoah/shenandoahCollectorPolicy.hpp

```diff
@@ -61,6 +61,8 @@ class ShenandoahCollectorPolicy : public CHeapObj<mtGC> {
   ShenandoahSharedFlag _in_shutdown;
   ShenandoahTracer* _tracer;
 
+  uint _degenerated_cycles_in_a_row;
```

Comment:
Nit, but we could call this `_consecutive_degenerated_gcs` for consistency with other fields in this class? (Also, possibly rename `_abbreviated_cycles` to `_abbreviated_gcs` for the same reason.

Reply:
Thanks.  Making this change.